src/channel.ts

```diff
@@ -52,12 +52,11 @@ export const ORDER = Channel.ORDER;
 export const OPACITY = Channel.OPACITY;
 
 export const CHANNELS = [X, Y, X2, Y2, ROW, COLUMN, SIZE, SHAPE, COLOR, ORDER, OPACITY, TEXT, DETAIL, LABEL];
-
-export const UNIT_CHANNELS = without(CHANNELS, [ROW, COLUMN]);
-export const UNIT_SCALE_CHANNELS = without(UNIT_CHANNELS, [X2, Y2, ORDER, DETAIL, TEXT, LABEL, X2, Y2]);
-export const SCALE_CHANNELS = UNIT_SCALE_CHANNELS.concat([ROW, COLUMN]);
-export const NONSPATIAL_CHANNELS = without(UNIT_CHANNELS, [X, Y, X2, Y2]);
-export const NONSPATIAL_SCALE_CHANNELS = without(UNIT_SCALE_CHANNELS, [X, Y]);
+export const UNIT_CHANNELS = [X, Y, X2, Y2, SIZE, SHAPE, COLOR, ORDER, OPACITY, TEXT, DETAIL, LABEL];  // CHANNELS without COLUMN, ROW
```

Comment:
I don't think this is the right way to fix correct type. It will lead to disastrous bugs that are hard to detect later on.  

I guess that you can just add as Channel[] and keep the same code.  (or try call `without<Channel>(...)`) or something.